usaspending_api/disaster/management/sql/disaster_covid19_file_f_grants.sql

```diff
@@ -26,10 +26,10 @@ SELECT
     "subaward_search"."funding_sub_tier_agency_na" AS "prime_award_funding_sub_agency_name",
     "subaward_search"."funding_office_code" AS "prime_award_funding_office_code",
     "subaward_search"."funding_office_name" AS "prime_award_funding_office_name",
-    (SELECT STRING_AGG (DISTINCT U2."tas_rendering_label", ';') AS "value" FROM "award_search" U0 LEFT OUTER JOIN "financial_accounts_by_awards" U1 ON (U0."award_id" = U1."award_id") LEFT OUTER JOIN "treasury_appropriation_account" U2 ON (U1."treasury_account_id" = U2."treasury_account_identifier") WHERE U0."award_id" = ("subaward_search"."award_id") GROUP BY U0."award_id") AS "prime_award_treasury_accounts_funding_this_award",
-    (SELECT STRING_AGG (DISTINCT U3."federal_account_code", ';') AS "value" FROM "award_search" U0 LEFT OUTER JOIN "financial_accounts_by_awards" U1 ON (U0."award_id" = U1."award_id") LEFT OUTER JOIN "treasury_appropriation_account" U2 ON (U1."treasury_account_id" = U2."treasury_account_identifier") LEFT OUTER JOIN "federal_account" U3 ON (U2."federal_account_id" = U3."id") WHERE U0."award_id" = ("subaward_search"."award_id") GROUP BY U0."award_id") AS "prime_award_federal_accounts_funding_this_award",
-    (SELECT STRING_AGG(DISTINCT CONCAT(U2."object_class", ':', U2.object_class_name), ';') FROM "award_search" U0 LEFT OUTER JOIN "financial_accounts_by_awards" U1 ON (U0. "award_id" = U1. "award_id") INNER JOIN "object_class" U2 ON (U1. "object_class_id" = U2. "id") WHERE U0. "award_id" = ("subaward_search"."award_id") and U1.object_class_id is not null GROUP BY U0. "award_id") AS "prime_award_object_classes_funding_this_award",
-    (SELECT STRING_AGG(DISTINCT CONCAT(U2."program_activity_code", ':', U2.program_activity_name), ';') FROM "award_search" U0 LEFT OUTER JOIN "financial_accounts_by_awards" U1 ON (U0. "award_id" = U1. "award_id") INNER JOIN "ref_program_activity" U2 ON (U1. "program_activity_id" = U2. "id") WHERE U0. "award_id" = ("subaward_search"."award_id") and U1.program_activity_id is not null GROUP BY U0. "award_id") AS "prime_award_program_activities_funding_this_award",
```

Comment:
Were these `ORDER BY` statements intended to be left in the PR? If so, why are the original SQL queries being updated?

Reply:
Please see Keith's comment. He requested we sort output to make it easier to compare new vs old. Apparently adding those order by to the string agg was the solution Clancy came up with.

Reply:
Apparently STRING_AGG supports ORDER BY as an argument https://www.postgresqltutorial.com/postgresql-aggregate-functions/postgresql-string_agg-function/.

Reply:
Gotcha, I see now that this only affects the outputs of those subqueries.